archinfo/arch_aarch64.py

```diff
@@ -134,7 +134,7 @@ def __init__(self, endness=Endness.LE):
                  general_purpose=True),
         Register(name='x30', size=8, subregisters=[('w30', 0, 4)], alias_names=('r30', 'lr'),
                  general_purpose=True),
-        Register(name='xsp', size=8, subregisters=[('wsp', 0, 4)], alias_names=('sp',),
+        Register(name='xsp', size=8, subregisters=[('wsp', 0, 4)], alias_names=('sp','xsp   '),
```

Comment:
Why the spaces in `'xsp    '`? Also, you missed a space in front of this string.